src/utils/isStandardSyntaxMediaFeature.js

```diff
@@ -16,5 +16,8 @@ export default function (mediaFeature) {
   // SCSS or Less interpolation
   if (hasInterpolation(mediaFeature)) { return false }
 
+  // SCSS variables
+  if (/\$.+?/.test(mediaFeature)) { return false }
```

Comment:
This is only test that there is a `$` *anywhere* in the string. Are we certain that that's a sufficient test?

Also, I wonder if we can be a little less restrictive, because I imagine a *lot* of people will want to use something like `(max-width: $foo)` and will be annoyed if the rule ignores that; whereas not so many people will want to use `(max-width: $foo + 2)`, and those people can always assign `$bar: $foo + 2;` to allow `(max-width: $bar)`. 

So: What if we just outlaw strings that contain variables *and* an operator?

Also: Won't this same issue apply to Less variables, so they should be targeted, as well?